config/build.yml

```diff
@@ -1,24 +1,24 @@
 #@ load("@ytt:data", "data")
 
 apiVersion: kbld.k14s.io/v1alpha1
-kind: Sources
+kind: Config
```

Comment:
Is this file should be reverted back to the original for it be mergable / tests to run